BN256G2/README.md

```diff
@@ -0,0 +1,18 @@
+# Transpiled [BN256G2.sol](https://github.com/musalbas/solidity-BN256G2) library.
```

Comment:
Could you add the warp version you used, as well as the Cairo version? Also, could it be possible to have the solidity performance info when used the same input on the solidity contract (like gas use and execution time, I believe it may be helpful for future re-transpilations with cairo 1).